odoo/http.py

```diff
@@ -696,6 +696,9 @@ def route(route=None, **routing):
     :param bool csrf: Whether CSRF protection should be enabled for the
         route. Enabled by default for ``'http'``-type requests, disabled
         by default for ``'json'``-type requests.
+    :param Callable[[Exception], Response] handle_access_error:
+        Implement a custom behavior if an error occurred when we retrieved the record
+        using to the model converter (access error or missing error).
```

Comment:
Users don't know what the model converter is, suggestion:
```suggestion
        Implement a custom behavior if an error occurred when we retrieved the record
        from the URL parameters (access error or missing error).
```

Reply:
wbu to rename the argument `handle_params_access_error` ?

Reply:
done